code/modules/client/preference/character.dm

```diff
@@ -822,6 +822,9 @@
 		else
 			preview_icon.Blend(rgb(-s_tone,  -s_tone,  -s_tone), ICON_SUBTRACT)
 
+	var/icon/hands_icon = icon(preview_icon)
+	hands_icon.Blend(icon('icons/mob/clothing/masking_helpers.dmi', "hands_mask"), ICON_MULTIPLY)
+
```

Comment:
Could you please explain what the mask is doing here? 

Also the masking helper sprite looks like it's for a human body only, is that going to be a problem when applied to other species?

Reply:
To my dismay when I was making this PR it turned out the lobby character preview _renders_ in a completely different way than ingame, this mask exists to cut out a handhole in the lobby character preview by "force".

It doesnt work on species, but neither does the entire lobby character preview, it puts human sprites on all species, and changing that to work with species sprites would be a particularly large undertaking compared to the current size of this PR, and not within the scope.